recipes/paleomix/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "paleomix" %}
+{% set version = "1.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/33/d5/7f9923dad69aaf6ba799d694f62d44c822350ef50042207b6ff8df235f95/{{ name }}-{{ version }}.tar.gz
+  sha256: fbfdcef6f0460b7afd2415aa957c6cacf56e4b4e9a8fda60fb63cac85c273496
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - adapterremoval
+    - samtools
+    - picard
+    - bowtie2
+    - bwa
+    - mapdamage2
+    - gatk
+    - coloredlogs
+    - pysam
+    - ruamel.yaml
+    - biopython
+    - setproctitle
+    - coverage
```

Comment:
The `setup.py` script specifies `coverage` as a `setup_requires` dependency, so it must be available at installation time. Translated into Conda terms, this means that you need to add it as a `host` dependency.

The full list of dependencies is at https://github.com/MikkelSchubert/paleomix/blob/v1.2.14/setup.py#L78 : 
`nose>=1.3.0`, `flexmock>=0.9.7`, `coverage>=4.0.0`

It’s nota good idea to have these development-only dependencies within `setup_requires`, and this has actually been fixed in the master version of paleomix. But to package 1.2.14, you’ll need to add the above to the host requirements. Or perhaps you can convince the author to release a new version shortly.